foundation-extensions/mobile-core/signals/README.md

```diff
@@ -37,8 +37,38 @@ import com.adobe.marketing.mobile.*;
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
-​ Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
+{% tab title="iOS (AEP 3.x)" %}
+​Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
```

Comment:
Can we provide a short explanation here, for example: The Signal extension should now be installed from the new AEPSignal pod, along with the its AEPCore dependency.